gnovm/pkg/gnolang/nodes.go

```diff
@@ -1209,6 +1209,7 @@ func ReadMemPackage(dir string, pkgPath string) (*std.MemPackage, error) {
 	allowedFiles := []string{ // make case insensitive?
 		"LICENSE",
 		"README.md",
+		"gno.mod",
```

Comment:
We need to sanitize the file for the chain (not from the current sanitization for CLI).

Sanitization for the chain involves removing elements that do not make sense on-chain but are valid off-chain. A simple example is the `replace` directive.

...Sanitization or a different Validation method to ensure that the file is compatible with on-chain requirements.